xml/admin_nfsganesha.xml

```diff
@@ -118,6 +96,28 @@
 </screen>
  </sect1>
 
+ <sect1 xml:id="ceph-nfsganesha-services">
+  <title>Starting or Restarting &ganesha;</title>
+  <para>
+   To start the &ganesha; service, run:
+  </para>
+ <screen>&prompt.cephuser;ceph orch start nfs.<replaceable>SERVICE_ID</replaceable></screen>
+  <para>
+   To restart the entire &ganesha; service, run:
```

Comment:
I suggest clarifying the difference between "starting the Ganesha service" and "restarting the **entire** Ganesha service"

Reply:
Ah hmm, @supriti is there a clarification to be made here?